packages/core/src/timeline.ts

```diff
@@ -103,6 +113,21 @@ export class Timeline {
             } else {
               await pluginResult;
             }
+
+            if ([PluginType.before, PluginType.enrichment].includes(type)) {
```

Comment:
Should we also notify for `after` plugins? maybe not as an `enriched` but as a special stage?